ert_gui/simulation/prefect_ensemble_experiment_panel.py

```diff
@@ -0,0 +1,88 @@
+from qtpy.QtWidgets import QFormLayout, QLabel
+
+from ert_gui.ertwidgets import addHelpToWidget
+from ert_gui.ertwidgets.caseselector import CaseSelector
+from ert_gui.ertwidgets.models.activerealizationsmodel import ActiveRealizationsModel
+from ert_gui.ertwidgets.models.init_iter_value import IterValueModel
+from ert_gui.ertwidgets.models.ertmodel import (
+    getRunPath,
+    get_runnable_realizations_mask,
+)
+from ert_gui.ertwidgets.stringbox import StringBox
+from ert_shared.ide.keywords.definitions import RangeStringArgument, IntegerArgument
+from ert_gui.simulation.simulation_config_panel import SimulationConfigPanel
+from ert_shared.models import EnsemblePrefectExperiment
+import yaml
+
+
+class PrefectEnsembleExperimentPanel(SimulationConfigPanel):
```

Comment:
Maybe prune this code a bit so that irrelevant parts are removed? (Seems a bit copypasta-ish)